src/alpaca_typer.erl

```diff
@@ -1501,13 +1501,13 @@ typ_of(Env, Lvl, #alpaca_map_add{line=L, to_add=A, existing=B}) ->
 
 %% Record typing:
 typ_of(Env, Lvl, #alpaca_record{members=Members}) ->
-    F = fun(#alpaca_record_member{name=N, val=V}, {Members, E}) ->
+    F = fun(#alpaca_record_member{name=N, val=V}, {ARMembers, E}) ->
                 case typ_of(E, Lvl, V) of
                     {error, _}=Err -> 
                         erlang:error(Err);
                     {VTyp, NextVar} ->
                         MTyp = #t_record_member{name=N, type=VTyp},
-                        {[MTyp|Members], update_counter(NextVar, E)}
+                        {[MTyp|ARMembers], update_counter(NextVar, E)}
```

Comment:
I renamed `Members` here to avoid shadowing the variable outside this fun, because it looks like that's what's expected/intended, but I didn't spend much time thinking about it, so this might be incorrect.